### DISCUSSION TITLE
attempting model selection on an essentially perfect fit is nonsenseWarning

### DISCUSSION THREAD

**Discussion by mosesnugraha** [teamalgoritma/community#594] [Q&A] [answered]
<img width="1440" alt="Screen Shot 2023-03-14 at 13 42 48" src="https://user-images.githubusercontent.com/123171973/224918579-110080a2-46a0-4e36-89b8-edd33d8a20e5.png">

selamat siang. Ini maksudnya apa ya? apakah tidak bisa menggunakan backward? atau miming modelnya yang salah?

**Comment by amaliasf** [comment, marked as answer]
Dear Pak @mosesnugraha 

Terkait error yang muncul saat melakukan `summary(model_backward)` dikarenakan pada chunck di atasnya ketika membuat model ada simbol ``` yang terhapus.  

Untuk pembuatan model seharusnya sudah berhasil hanya menampilkan pesan warning saja. Bisa dicek kembali apakah ada yang janggal dari summary model yang dihasilkan atau tidak.

Semoga menjawab Pak ^^ Silahkan kalau ada pertanyaan lanjutan.
